diff --git a/libpbe/src/SmtpClient.cc b/libpbe/src/SmtpClient.cc
index 0a4933d..b16ebf9 100644
--- a/libpbe/src/SmtpClient.cc
+++ b/libpbe/src/SmtpClient.cc
@@ -15,6 +15,7 @@
 // You should have received a copy of the GNU General Public License
 // along with this program; if not, write to the Free Software
 // Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#include <stdio.h>
 #include "SmtpClient.hh"
 
 #include "ip.hh"
